cirq/experiments/fidelity_estimation.py

```diff
@@ -470,17 +505,42 @@ def __enter__(
     def __exit__(self, exc_type, exc_val, exc_tb):
         pass
 
-    def update(self, increment: int):
+    def update(self, n: int = 1):
```

Comment:
Why obscure this?

Reply:
this matches the tqdm signature if you wanted to use keyword arguments

Reply:
One more reason to wrap tqdm api in our own api? We could then even swap it for alternative implementations....

Reply:
I'm not sure using `n` instead of `increment` adds all that much impetus to (re)wrap tqdm, a very popular progress bar library :)

As it exists, you can swap in alternative implementations. One such example is given here!